guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

```diff
@@ -134,7 +134,13 @@ Such servers lie outside of the {Project} infrastructure, however, it is recomme
 {Project} provides features for precise management of the content life cycle.
 A *life cycle environment* represents a stage in the content life cycle, a *Content View* is a filtered set of content, and can be considered as a defined subset of content.
 By associating Content Views with life cycle environments, you make content available to hosts in a defined way.
-See xref:figu-Content_Life_Cycle_{context}[] for visualization of the process.
+
+ifndef::satellite[]
+include::../common/modules/snip_red-hat-images.adoc[]
+endif::[]
+
+image::life-cycle-satellite.png[Content Life Cycle in {ProjectName}]
```

Comment:
```suggestion
image::life-cycle-satellite.png[Content Lifecycle in {ProjectName}]
```

At other places, we might also talk about the "Application Lifecycle". Reword?

Reply:
I'd rather leave this untouched for now. The Deployment Planning part is out of scope of this PR.

Regarding life cycle vs lifecycle, this part of the guide uses "life cycle", so I think it's best to stay consistent for now -- until I have time to review Deployment Planning in another PR.

Reply:
I think this has been changed globally by Brian since my comment. See 4a14a77f1f0ba305303602fc62be1220c1b7c6e8

Reply:
Ok, thank you Brian! I found 2 remaining instances of "Life cycle" so I fixed those.

But I'll leave the question of application vs content life cycle for later when I start looking at the Deployment part of the guide.